app/_gateway_entities/target.md

```diff
@@ -23,11 +23,16 @@ schema:
     api: gateway/admin-ee
     path: /schemas/Target
 
+api_specs:
+    - gateway/admin-oss
+    - gateway/admin-ee
+    - konnect/control-planes-config
+
 ---
 
 ## What is a Target?
 
-A Target is an IP address/hostname with a port that identifies an instance of a backend service.
+A Target identifies an instance of a service application using an IP address or hostname with a port.
```

Comment:
```suggestion
A Target identifies an instance of a service application using an IP address or hostname with a port. A service application is an application you own that serves requests and responses.
```
I think what was missing for me was defining what a service application is with an example of sorts. I don't know if my rewrite is accurate.

Reply:
For this page specifically, instead of defining the term, what if we updated the diagram to contain the term so it's clearer what we mean.   Right now the box on the far right in the diagram has the term "upstream targets", but those are the "service applications". It would be cool if the diagram could show that the service applications can be on different hosts, networks, etc... Right now they are just shown in a single box.

Reply:
Ah, I missed those, nice catch.

Reply:
Thanks! Same w/ the Upstreams page as well